### ISSUE TITLE
[Extension] Manifest v3

### ISSUE THREAD

**Issue by tombrunet** [IBMa/equal-access#311]
Chrome is pushing changes to extensions in an effort rolled up as 'Manifest V3'. Much of the pushback to MV3 is that it will disable ad-blockers. However, an aspect that affects us is that it advertises to disable all forms of remote code execution. This affects our extensions because we do not currently package the rules - they are fetched when a particular archive is selected. By fetching the rules, we can ensure that all users are using the same version of the rules - and avoids conflicts across tools. Without fetches, we found users would get confused by conflicting results due to a lack of tool updates.

At the moment MV3 is just starting to roll out, ~and MV2 end-of-life is still TBD~  (see update on mv2 EOL in comments below): https://web.archive.org/web/20201130053006/https://developer.chrome.com/extensions/migrating_to_manifest_v3#timeline

Another concern is that Firefox may or may not implement MV3 in the same way, so we may run into some compatibility issues across the browsers.

Some thoughts on how we might still be able to operate with MV3:
* [ ] Include all archives with the extension
* [ ] Will need a process to phase out archives so that the extension size isn't 'too big'
* [ ] Will need to ping the server for archive version, and then notify the user that their extension is outdated and needs to be updated.

**Comment by marcjohlic** [comment]
Mv2 timeline update:   https://developer.chrome.com/docs/extensions/mv3/mv2-sunset/

EOL June 2023 - Devs can no longer update Mv2 Jan 2023

![image](https://user-images.githubusercontent.com/6586306/151867093-717be34e-1a1c-40c4-87ca-44993583f22a.png)

**Comment by tombrunet** [comment]
It looks like we’ll have to package the engines with the extensions. That will make updates harder and more likely to get out of sync with other tools (particularly when using latest/preview). It doesn’t look like there’s much we can do about that
The exceljs package appears to have an eval in it that crashes us. Going to need to dig into that further.
I was able to get a scan to run, but I disabled a bunch of code while hunting down the exceljs eval, so not clear what else will be problematic yet.

I also did a first pass at updating to Carbon 11 due to hotload issues, and the build times are down from 2 minutes to 10 seconds in watch mode, which makes it tolerable. There are some layout cleanup issues I still need to address.
